src/details/ArborX_DetailsSortUtils.hpp

```diff
@@ -85,9 +85,21 @@ sortObjects(ExecutionSpace const &space, ViewType &view)
 
   Kokkos::MinMaxScalar<ValueType> result;
   Kokkos::MinMax<ValueType> reducer(result);
-  parallel_reduce("ArborX::Sorting::find_min_max_view",
-                  Kokkos::RangePolicy<ExecutionSpace>(space, 0, n),
-                  Kokkos::Impl::min_max_functor<ViewType>(view), reducer);
+  parallel_reduce(
+      "ArborX::Sorting::find_min_max_view",
+      Kokkos::RangePolicy<ExecutionSpace>(space, 0, n),
+      KOKKOS_LAMBDA(int i, Kokkos::MinMaxScalar<ValueType> &local_minmax) {
+        auto const val = view(i);
+        if (val < local_minmax.min_val)
+        {
+          local_minmax.min_val = val;
+        }
+        if (val > local_minmax.max_val)
+        {
+          local_minmax.max_val = val;
+        }
+      },
+      reducer);
```

Comment:
I'm confused. If we have `minMax` in `ArborX_DetailsUtils.hpp`, can't we use it here?

Reply:
Sure will do but I will then have to handle explicitly empty views.